dev/releases/README.md

```diff
@@ -17,86 +41,53 @@ Before starting the release process, the scripts have the following dependencies
+      - Writes a file `MANIFEST` which contains a list of all files to be uploaded.
+   3. Run `make_github_release.py vX.Y.Z tmp/` from the GAP root directory; `tmp/` is the path to the temporary directory created by `make_archives.py`.
       - Creates the release on GitHub which matches the tag.
-      - Uploads the archives listed in the `MANIFEST` file as assets.
-4. Change to the root directory of your clone of the `gap-system/GapWWW` repository. 
-   This should be obvious why.
-5. Run `update_website.py` in there.
-   - Fetches the release assets, extracts and configures/builds GAP in a tmp directory.
-   - Extracts info from the built and rewrites various YAML files.
-   - Extracts info about packages and updates YAML files.
-   - Commits, pushes and creates pull request to GapWWW.
-6. [optional] Remove the `tmp` directories (in GapWWW and gap directories).
+      - Uploads the archives listed in the `MANIFEST` file as assets, along with a corresponding `.sha256` file.
+   4. Create some Windows installers and uploads them to the GitHub release, too. A Windows computer is required for this step.
+5. Access your local clone of the `GapWWW` repository.
+6. Make sure that your clone is up to date with `gap-system/GapWWW`.
+7. Run `update_website.py` in there (see `update_website.py --help`). This:
+   - Fetches the GitHub releases data from `gap-system/gap`.
+   - Deletes, modifies, and adds various JSON and HTML files according to this data.
+8. Inspect and commit the changes, and update the website as documented on `gap-system/GapWWW`.
 
 
 ## GitHub access token
+<a name="github-access-token"></a>
 
-The `make_github_release.py` script needs limited write access to the GAP repository
-in order to upload the release archives for you. In order to do this, the
-scripts needs to authenticate itself with GitHub, for which it needs a
-so-called "personal access token". You can generate such a token as follows
-(see also <https://help.github.com/articles/creating-an-access-token-for-command-line-use>).
+Various scripts in the same folder as this README need limited read/write access to the GAP repository in order to interact with the GitHub API, including to create a release and upload release archives. In order to do this, the scripts need to authenticate with GitHub, for which it needs a so-called "personal access token".
+You can generate such a token as follows (see also <https://help.github.com/articles/creating-an-access-token-for-command-line-use>).
 
 1. Go to <https://github.com/settings/tokens>.
-
 2. Click **Generate new token**.
-
 3. Select the scope "public_repo", and give your token a descriptive name.
-
 4. Click **Generate token** at the bottom of the page.
+5. Copy the token to your clipboard. For security reasons, after you navigate off the page, you will not be able to see the token again. You therefore should store it somewhere, e.g. with option 3 in the following list.
```

Comment:
What's "option 3 in the following list"? I guess that list doesn't exist anymore.